mypy/errors.py

```diff
@@ -199,9 +204,9 @@ class Errors:
     seen_import_error = False
 
     # Error baseline
-    baseline: dict[str, list[ErrorInfo]] = {}
+    baseline: Dict[str, List[ErrorInfo]] = {}
     # All detected errors before baseline filter
-    all_errors: dict[str, list[ErrorInfo]] = {}
+    all_errors: Dict[str, List[ErrorInfo]] = {}
```

Comment:
arent `Dict` and `List` deprecated

Reply:
mypy is expected to be used with python 3.6